lib/rules/no-html-safe.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const ERROR_MESSAGE = 'Do not use htmlSafe.';
```

Comment:
Error message is a bit dry. Would it better to do something like or something similar?

```
Do not use htmlSafe. It can open you up to XSS vulnerabilities.
```

Reply:
`Do not use htmlSafe.` is fine. I normally like to use the format `Use X instead of Y` for error messages, but there doesn't seem to be one single preferred fix for this violation.

Reply:
Yeah I like doing similar but not sure we could fit any actionable advice in the error. How about linking to rule .md file?

Reply:
We should not link to the documentation in the error message because the rule already includes the documentation link in its meta data and many IDEs will already provide a link to the documentation directly on the highlighted rule violation.